Kickstarter-iOS/Features/PledgeView/Controllers/NoShippingPledgeViewControllerTests.swift

```diff
@@ -170,10 +170,56 @@ final class NoShippingPledgeViewControllerTests: TestCase {
       }
   }
 
+  func testView_ShowsEstimatedShippingView() {
+    let shippingRule = ShippingRule.template
+      |> ShippingRule.lens.estimatedMin .~ Money(amount: 5.0)
+      |> ShippingRule.lens.estimatedMax .~ Money(amount: 10.0)
+    let reward = Reward.template
+      |> Reward.lens.shipping.enabled .~ true
+      |> Reward.lens.shippingRules .~ [shippingRule]
+      |> Reward.lens.id .~ 99
+    let mockConfigClient = MockRemoteConfigClient()
+    mockConfigClient.features = [
+      RemoteConfigFeature.noShippingAtCheckout.rawValue: true
+    ]
+
+    orthogonalCombos(Language.allLanguages, [Device.phone4_7inch, Device.pad], [nil, User.template])
+      .forEach { language, device, _ in
```

Comment:
Since we're not using the user in this test. can you delete it from the `forEach`?